app/routes/photo-albums/photo-album/photos/photo.js

```
@@ -1,7 +1,11 @@
 import { ApplicationRoute } from 'amber-ui/routes/application/application';
 
 export default class PhotoRoute extends ApplicationRoute {
-  queryParams = {};
+  queryParams = {
+    includeOldMembers: {
+      replace: true
```

ESLint

[error] Insert `,`

(prettier/prettier)